src/pipeline/pipeline.ml

```diff
@@ -230,7 +230,12 @@ let check_builtin what src senv0 : Syntax.prog * stat_env =
 let prelude, initial_stat_env0 =
   check_builtin "prelude" Prelude.prelude Typing.initial_scope
 let internals, initial_stat_env =
-  check_builtin "internals" Prelude.internals initial_stat_env0
+  let aliasing = !Flags.experimental_field_aliasing in
```

Comment:
I don't like the use of the aliasing feature in prod. Let's try to remove that, if only to keep pipeline as simple as possible